client/components/email-verification/email-verification-dialog/index.jsx

```diff
@@ -46,38 +49,37 @@ class VerifyEmailDialog extends Component {
 			>
 				{ this.getResendButtonLabel() }
 			</Button>,
-			<Button key="close" primary onClick={ this.handleClose }>
-				{ this.props.translate( 'OK' ) }
-			</Button>,
 		];
 	}
 
 	render() {
 		const strings = {
-			confirmHeading: this.props.translate( 'Please verify your email address:' ),
+			confirmHeading: this.props.translate( 'Verify your email' ),
 
 			confirmExplanation: this.props.translate(
-				'When you first signed up for a WordPress.com account we sent you an email. ' +
-					'Please open it and click on the blue button to verify your email address.'
+				"Check your inbox at {{wrapper}}%(email)s{{/wrapper}} for the confirmation email, or click 'Resend Email' to get a new one.",
```

Comment:
I wonder if now would be a good time to ensure that the sentence casing on the existing button is updated (ie. `Resend Email` -> `Resend email`). I suppose that this button was missed during the work in #38231, but might make sense to fix now before this new string is translated?

Reply:
> Resend Email -> Resend email

Good point. @javierarce, does that make sense to you?

Reply:
Updated. Thanks Aurorum!